src/blocktypes/containers.nix

```diff
@@ -35,79 +67,46 @@
       (mkCommand currentSystem {
         name = "publish";
         description = "copy the image to its remote registry";
-        command = let
-          image = target.imageRefUnsafe or "${target.imageName}:${target.imageTag}";
-        in ''
-          # docker://${builtins.unsafeDiscardStringContext image}
-          ${target.copyToRegistry}/bin/copy-to-registry
+        command = ''
+          ${copyTags}
+          copy_tags docker://${img}
         '';
-        proviso =
-          l.toFile "container-proviso"
-          # bash
-          ''
-            function proviso() {
-            local -n input=$1
-            local -n output=$2
-
-            local -a images
-            local delim="$RANDOM"
-
-            function get_images () {
-              command nix show-derivation $@ \
-              | command jq -r '.[].env.text' \
-              | command grep -o 'docker://\S*'
-            }
-
-            drvs="$(command jq -r '.actionDrv | select(. != "null")' <<< "''${input[@]}")"
-
-            mapfile -t images < <(get_images $drvs)
-
-            command cat << "$delim" > /tmp/check.sh
-            #!/usr/bin/env bash
-            if ! command skopeo inspect --insecure-policy "$1" &>/dev/null; then
-            echo "$1" >> /tmp/no_exist
-            fi
-            $delim
-
-            chmod +x /tmp/check.sh
-
-            rm -f /tmp/no_exist
-
-            echo "''${images[@]}" \
-            | command xargs -n 1 -P 0 /tmp/check.sh
-
-            declare -a filtered
-
-            for i in "''${!images[@]}"; do
-              if command grep "''${images[$i]}" /tmp/no_exist &>/dev/null; then
-                filtered+=("''${input[$i]}")
-              fi
-            done
-
-            output=$(command jq -cs '. += $p' --argjson p "$output" <<< "''${filtered[@]}")
-            }
-          '';
+        proviso = l.toFile "container-proviso" ''
+          SKOPEO=${./proviso/skopeo.sh}
+          ${builtins.readFile ./proviso/publish.sh}
+        '';
+        meta.tags = map (tag: "${img}:${tag}") tags;
       })
       (mkCommand currentSystem {
-        name = "copy-to-registry";
-        description = "copy the image to its remote registry";
+        name = "load";
+        description = "load image to the local docker daemon";
         command = ''
-          ${target.copyToRegistry}/bin/copy-to-registry
+          ${copyTags}
+
+          if command -v podman &> /dev/null; then
+             ixecontainerho "Podman detected: copy to local podman"
```

Comment:
`ixecontainerho`? :smile:

Reply:
you wrote that in the first commit here so I thought it was intentional. Typo maybe?

Reply:
Yeah, oops :-)